src/commands/import.js

```diff
@@ -22,21 +25,54 @@ class ImportCommand extends TerraformCommand {
   run() {
     const configContentArr = this.getOption('config');
     const providerContent = this.getOption('provider');
+    const batch = this.getOption('batch');
+    const include = this.getOption('include');
```

Comment:
please rethink this functionality to reuse `--include`, `--exclude`, `--include-regex` and `--exclude-regex` defined in `terraform-command.js` (ref: https://github.com/TerraHubCorp/terrahub/blob/dev/src/terraform-command.js#L43)

Reply:
will be addressed in future iteration of this feature